buildpack.md

```diff
@@ -704,7 +704,9 @@ Given the start command and execution strategy,
 
 1. The lifecycle MUST set all buildpack-provided launch environment variables as described in the [Environment](#environment) section.
 
-2. The lifecycle MUST
+2. The lifecycle MUST set the working directory for the process to `working-directory`, or to the application directory if `working-directory` is not specified.
```

Comment:
I think we need to explicitly call out that the working directory applies to the process [but not to `exec.d` or `profile.d` launch helpers](https://github.com/buildpacks/rfcs/blob/main/text/0081-process-specific-working-directory.md#how-it-works).

Reply:
Sounds good - I also moved this bullet further down so that it is chronologically accurate.